archinstall/lib/installer.py

```diff
@@ -34,7 +34,8 @@ class Installer():
 	:type hostname: str, optional
 
 	"""
-	def __init__(self, target, *, base_packages='base base-devel linux linux-firmware efibootmgr'):
+	def __init__(self, target, *, base_packages=' base base-devel linux-firmware efibootmgr', kernel='linux'):
```

Comment:
This should be changed to be kernels (plural) and accept an array instead of a string, so that you can do ['linux', 'linux-lts']. The user should be able to select one at boot, and each should be given a bootloader entry for both GRUB and systemd-boot depending on installation type.

Reply:
I believe it should be fine to have multiple kernels without any changes to the GRUB installation setup as `grub-mkconfig` should pick up other kernels automatically

Reply:
Systemd-boot needs an entry for each though

Reply:
I'm creating an issue for this so we don't forget about this discussion @dylanmtaylor